content/v2.0/reference/release-notes/flux.md

```diff
@@ -16,6 +16,121 @@ Though newer versions of Flux may be available, they will not be included with
 InfluxDB until the next InfluxDB v2.0 release._
 {{% /note %}}
 
+## v0.68.0 [2020-05-28]
+This version of Flux introduces an updated type inference system that improves
+performance, error messaging, and usability of the
+[Flux Language Server Protocol (LSP)](https://github.com/influxdata/flux-lsp).
+
+## Breaking Changes
+- Change signature of `group()` function.
+
+## Features
```

Comment:
being the list is so long, maybe we can add grouping for scanability:

**New functions**
- Add [`fieldKeys()`](/v2.0/reference/flux/stdlib/influxdb-v1/fieldkeys/) and
  [`measurementFieldKeys()`](/v2.0/reference/flux/stdlib/influxdb-v1/measurementfieldkeys/)
  to v1 package.
- Add [`findColumn()`](/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/findcolumn/)
  and [`findRecord()`](/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/findrecord/) functions.
- Add `experimental.chain()` function.
- Add `lookuptype` function for stdlib builtins.

**Function updates**
- Make `Eval()` and `EvalAST()` use libflux for parsing and analysis.
- Return `false` if `contains()` is called with an empty set.
- Support remote `buckets()` and `v1.databases()` calls.

**Package updates**
- Handle multi-file packages.
- Add support for `pkg-config`.
- Add Snowflake support to SQL package.

**Node updates**
- Transform semantic nodes back to AST nodes.
- Add semantic nodes for bad statement and bad expression.


**Other features**
- Add a context to `plantest.RuleTestCase`.
- Add performance optimizations.
- Add a dynamically linked Valgrind test.
- Add location information to type error messages.
- Add all Linux cross-compilation tools to release Docker image.
- Add support for static linking.
- Add `influxdb` source.

Reply:
I thought about this, but it breaks convention from all our other release notes.